app/controllers/users.js

```diff
@@ -22,3 +22,14 @@ exports.signIn = async ({ body: { email, password } }, res, next) => {
     return next(error);
   }
 };
+
+exports.getUsers = ({ query }, res, next) =>
+  usersService
+    .getPaginatedUsers(query)
+    .then(paginatedUsersInfo => {
```

Comment:
paginatedUsersInfo is confusing, it does not read by itself, I need to go to the service to understand what it is. Try to use destructuring here, something like: `{ count, size, page, users }` in order to be more declarative.